### ISSUE TITLE
Questie sometimes doesn't detect quests being accepted with autoaccept feature.

### ISSUE THREAD

**Issue by Cabro** [Questie/Questie#6418]
This happens in the case of turning in a quest with a followup from the same NPC. Sometimes, the autoaccept feature is too fast for Questie's quest accept detection.

![image](https://github.com/user-attachments/assets/2c7abcbb-3426-48c1-807e-ca2ad953619b)
![image](https://github.com/user-attachments/assets/4f52d159-1247-468d-a63e-1581ec2cb80a)

Yes, this is after a new mouseover.

**Comment by BreakBB** [comment]
Were you using https://github.com/Questie/Questie/tree/auto-questing when this happened?

**Comment by Cabro** [comment]
No, i reverted back to default since... that one wasn't finished? :)

**Comment by BreakBB** [comment]
Correct. Good to know and absolutely something that we need to check when continue reworking auto questing.
